server/userNotification/create.ts

```diff
@@ -93,6 +93,7 @@ const createNotificationsForThreadComment = async (
 				activityItemId: item.id,
 			};
 		}),
+		{ individualHooks: true },
```

Comment:
I'm _pretty_ sure I removed this by mistake. I came across this bug/"feature" in other places as well, I remember adding this somewhere, but don't think I ever intentionally removed it. Very sorry!

Reply:
not a problem! thanks for adding the test!